app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue

```diff
@@ -1,107 +1,103 @@
 <script>
 import { mapGetters } from 'vuex';
 import { useAccount } from 'dashboard/composables/useAccount';
+import OnboardingFeatureCard from './OnboardingFeatureCard.vue';
 
 export default {
+  components: {
+    OnboardingFeatureCard,
+  },
   setup() {
     const { accountScopedUrl } = useAccount();
     return {
       accountScopedUrl,
     };
   },
   computed: {
-    ...mapGetters({ globalConfig: 'globalConfig/get' }),
-    newInboxURL() {
-      return this.accountScopedUrl('settings/inboxes/new');
+    ...mapGetters({
+      globalConfig: 'globalConfig/get',
+      currentUser: 'getCurrentUser',
+    }),
+    greeting() {
+      const time = new Date();
+      const hours = time.getHours();
+      if (hours < 12) return this.$t('ONBOARDING.MORNING');
+      if (hours < 18) return this.$t('ONBOARDING.AFTERNOON');
+      return this.$t('ONBOARDING.EVENING');
+    },
+    greetingMessage() {
+      return this.$t('ONBOARDING.GREETING', {
+        greeting: this.greeting,
+        name: this.currentUser.name,
+        installationName: this.globalConfig.installationName,
+      });
```

Comment:
This is still problematic. The sentence is broken. You should not pass anything other than numbers, names as variables. Here `Good morning / evening` is passed as a variable to the sentence. Please read this guideline properly. 
https://www.chatwoot.com/hc/handbook/articles/1710791123-translation-guidelines

Reply:
@pranavrajs Updated, how does this look?